homeassistant/components/binary_sensor/hue.py

```diff
@@ -0,0 +1,338 @@
+"""
+This component provides binary sensor support for the Philips Hue system.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.hue/
+"""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+
+from aiohue.sensors import (TYPE_CLIP_GENERICFLAG, TYPE_CLIP_OPENCLOSE, TYPE_CLIP_PRESENCE,
+                            TYPE_DAYLIGHT, TYPE_ZLL_PRESENCE)
+import homeassistant.components.hue as hue
+from homeassistant.components.hue.const import (ATTR_LAST_UPDATED, ICON_DAY, ICON_NIGHT)
+from homeassistant.const import ATTR_BATTERY_LEVEL
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+DEPENDENCIES = ['hue']
+SCAN_INTERVAL = timedelta(seconds=1)
```

Comment:
1 second scan interval? That seems too short.

Reply:
It is short, but with a polling API and remotes / motion sensors, you need to poll aggressively or else a button press or motion sense won't trigger an automation until the next polling interval and if that's too long then there isn't much point.